R/uvhydrograph-render.R

```diff
@@ -167,6 +167,72 @@ createSecondaryPlot <- function(data, month){
       status_msg <- paste('Corrected data missing for', data$reportMetadata$secondaryParameter)
     }
   } 
+
+  isUpchainSeries <- any(grepl("downsampledUpchainSeries", names(data)))
```

Comment:
I think you could just pass both values in as a vector since it's determining max/min of data. We want these to be on the same plot (that's what I gathered from the ticket)

```r
secCorr <- c(secondaryData$corr_UV_ref$value, secondaryData$corr_UV_up$value)
secUncorr <- c(secondaryData$uncorr_UV_ref$value, secondaryData$uncorr_UV_up$value)
YAxisInterval(secCorr, secUncorr)
```